nodes/widgets/locales/en-US/ui_table.html

```diff
@@ -0,0 +1,49 @@
+<script type="text/html" data-help-name="ui-table">
+    <p>
+        Display and manage complex datasets with ease using the ui-table widget.
+    </p>
```

Comment:
Ideally, help would explain what the input data shape should be?

Reply:
Resolved in https://github.com/FlowFuse/node-red-dashboard/pull/873/commits/df1af51d3d29e3cabfbac018a9d85f9721a45afa